ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -470,6 +474,7 @@ const TransactionSummaryScreen = (): React.ReactElement => {
             () => continueWithPayment(),
             () => startAssistanceRequest(error, messageId)
           )}
+          <VSpacer />
```

Comment:
I think it is better to make the size explicit, just to avoid problems in any future changes to the default value. What do you think?

```suggestion
          <VSpacer size={16} />
```

Reply:
Addressed into 8ec2022e573aef51405c0196666f42f3c4146ac7